src/bun.js/bindings/BunString.cpp

```diff
@@ -196,6 +196,22 @@ BunString toStringRef(WTF::StringImpl* wtfString)
     return { BunStringTag::WTFStringImpl, { .wtf = wtfString } };
 }
 
+BunString toStringView(StringView view) {
+    view.is8Bit();
+    return {
+        BunStringTag::StaticZigString,
+        { .zig = {
+            .ptr = (const LChar*)(view.is8Bit()
+                ? (size_t)view.rawCharacters()
+                : ((size_t)view.rawCharacters() + (static_cast<uint64_t>(1) << 63))
+            ),
+            .len = view.length()
+        } }
+    };
+}
```

Comment:
If `StringView` does not contain a `WTF::StringImpl` inside, then it should be `ZigString` so that we do not attempt to create a string which is never cloned/freed

Reply:
i chose `StaticZigString` as the tag to specifically avoid any possibility of freeing the data. not sure if this is copied. going on the single usage, i dont think it is.

A StringView is a string view. It doesn't have a destructor or a ref to the underlying string in release builds.

<img width="737" alt="image" src="https://github.com/oven-sh/bun/assets/24465214/0033d0e4-8b1e-4ba3-a0d5-3e6144d5d996">